docs/catalog/cloud/match-slo.md

```diff
@@ -0,0 +1,50 @@
+---
+version: 1.0
+submitted_by: danielmeixner
+published_date: TBD
+category: cloud
+tags: 
+ - [PATTERN_TAGS]
```

Comment:
Please add relevant tags to this list. I would suggest: 
- Cloud
- Reliability/Availability
- Monitoring